packages/testing/src/index.ts

```diff
@@ -134,6 +134,13 @@ export function createTestkit(
 
       return r;
     },
+    executeRaw: makeExecute(async (args: ExecutionArgs) => {
+      const proxy = initRequest();
+      return await proxy.execute({
+        ...args,
+        contextValue: await proxy.contextFactory(args.contextValue),
+      });
+    }),
```

Comment:
@dotansimha At the moment the testkit is pretty helix tailored. I assume this is in order to be more integration test-like.

I added this method here for now, but maybe we should in that case just construct all the stuff in place within the test?

Reply:
I think we can remove helix for testing, I initially used it because I wanted to test envelop with a request, but it's no longer needed.